### PR TITLE
Chemistry

### PR DESCRIPTION
**Pull request by Zyzarda**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
add: Added Sulfur Trioxide, a potent smoke that is less effective than the usual recipe but is acidic in nature. 3 parts water, 1 parts sulfur, 1 parts sulphuric acid. 250k to complete the recipe, reacts on creation.
/:cl:

## About The Pull Request
Adds a different kind of smoke producing recipe that is much less effective than the regular smoke recipe.

## Why It's Good For The Game
Chemistry needs more variety

### CODE REVIEW

File: hippiestation/code/datums/martial/armstrong.dm [hunk 1]
```diff
@@ -299,7 +299,6 @@ var/horse_stance_effects = FALSE // ensures the horse stance gains it effect
 		return 1
 	else // Prevents you from comboing dead lads, returns the default behavior.
 		return
-
```
Comment (Manly-Dwarf) [inline hippiestation/code/datums/martial/armstrong.dm:302]: 👺

Comment (Zyzarda) [reply hippiestation/code/datums/martial/armstrong.dm:302]: btfo dwarffag

**Comment by yoyobatty** [comment]
Interesting, sulfur trioxide SO3 is a precursor to sulfuric acid H2SO4, might be worth changing the formula.

**Review by Jujumatic — DISMISSED** [review]

**Comment by Jujumatic** [comment]
needs head input

File: hippiestation/code/datums/martial/armstrong.dm (hunk 1 above)
Comment (k-k-karma) [reply hippiestation/code/datums/martial/armstrong.dm]: remove this

File: hippiestation/code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm [hunk 2]
```diff
@@ -336,3 +336,24 @@
...
+	description = "Makes a large cloud of sulphuric acid."
+	reagent_state = LIQUID
+	metabolization_rate = 3
+	color = "#FF7F50"
+	taste_description = "burning copper"
+/datum/reagent/sulfur_trioxide/reaction_mob(mob/living/carbon/C, method=TOUCH, reac_volume)
+	if(!istype(C))
+		C.adjustBruteLoss(0.1 * min(0.2 * reac_volume))
+	..()
+	return TRUE
```
Comment (k-k-karma) [inline hippiestation/code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: what the fuck is this?

Comment (Zyzarda) [reply hippiestation/code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm:351]: Copypasted code that I might've forgotten to remove a bit.

**Comment by Zyzarda** [comment]
i've absolutely fucked up and misclick-deleted armstrong style on my git, looking into fixing this

**Pull request closed without merging**